common-app-lib/src/cmr/common_app/services/kms_lookup.clj

```diff
@@ -140,16 +140,37 @@
   [kms-index location-string]
   (get-in kms-index [:locations-index (str/upper-case location-string)]))
 
-(defn lookup-by-umm-c-keyword
+(defn- remove-long-name-from-kms-index
```

Comment:
Add a docstring. Besides just explaining what it does, why we have this function would be useful to document.